sdk/servicebus/service-bus/samples-v1/javascript/README.md

```diff
@@ -0,0 +1,86 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-service-bus
+urlFragment: service-bus-javascript
+---
```

Comment:
@willmtemple, is this enough to get the samples published?
One thing to note - the name of this folder is `samples-v1`, would that be a problem?

Reply:
This should do the trick. You may want to use a different `urlFragment` here to prevent it from colliding once the next set of samples are published.

Reply:
When the next set gets published, I assume we would want to replace the samples with the new ones. So, I think there is no need for another `urlFragment`.
@ramya-rao-a thoughts?

Reply:
> When the next set gets published, I assume we would want to replace the samples with the new ones

Yes.